app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -294,9 +300,13 @@ object TelemetryWrapper {
     }
 
     private var histogram = IntArray(HISTOGRAM_SIZE)
+    private var domainMap = HashSet<String>()
+    private var numUri = 0
 
     @JvmStatic
-    fun addLoadToHistogram(newLoadTime: Long) {
+    fun addLoadToHistogram(url: String, newLoadTime: Long) {
```

Comment:
Looks like we're storing this url in order to count distinct values, but we only upload the count.

@ekager can you add a comment that the keys of this map should never be uploaded to telemetry?